src/chart/XYAxis.js

```diff
@@ -63,18 +63,18 @@
 
         var test = element.append("g");
 
-        var svgXAxis = test.append("g")
+        var svgDataAxis = test.append("g")
             .attr("class", this.orientation() === "horizontal" ? "x axis" : "y axis")
             .attr("transform", "translate(0," + height + ")")
-            .call(this.orientation() === "horizontal" ? this.xAxis : this.yAxis)
+            .call(this.orientation() === "horizontal" ? this.dataAxis : this.columnAxis)
         ;
-        var svgYAxis = test.append("g")
+        var svgColumnAxis = test.append("g")
```

Comment:
Same for this:   This is not a "columnAxis" as it sometimes contains columns. It is an YAxis...